test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,24 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // all objects are not null
+        assertNotNull("abc", "ab");
+
+        // one of the object is null
+        assertHasNull("abc", null, "bc");
+        assertHasNull((Object) null);
+    }
+
+    private void assertNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
+    private void assertHasNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
```

Comment:
It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value.
In assertAreUnique and assertNotUnique refactoring helps to convert arrays passed to the method as collections.
For isAnyNull refactoring just adds on to the height of the code.